git/git.go

```diff
@@ -223,6 +223,10 @@ func Log(sha1, sha2 string) (string, error) {
 	return outputs, nil
 }
 
+func Push(remote, source, dest string) error {
+	return Spawn("push", remote, fmt.Sprintf("%s:%s", source, dest))
```

Comment:
This is only being used in a single place in the codebase. I don't  think it warrants its own method. I would just inline the `git.Spawn` invocation in `pull-request.go`